### PR TITLE
[TW-1487] Doc Improve Schema Documentation with Live Preview

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Adds a new section about live preview for schema docs
- Adds a note in viewing schema docs section linking back to live preview section

View the beta docs here: [Previewing schema documentation](https://tw-1487-schema-live-preview.learning.postman-beta.com/docs/designing-and-developing-your-api/developing-an-api/defining-an-api/#previewing-schema-documentation)

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md [hunk 1]
```diff
@@ -97,18 +98,29 @@ To edit an API definition, select an API in the sidebar to expand it, then selec
...
 In the upper right of the API definition editor are options to beautify the content, wrap text, copy, change the file format, and search. When you finish editing your API definition, select **Save**.
 
-<img alt="API Builder editing options" src="https://assets.postman.com/postman-docs/v10/api-builder-edit-options-v10.jpg"/>
+<img alt="API Builder editing options" src="https://assets.postman.com/postman-docs/v10/api-builder-edit-options-v10-21.jpg"/>
 
 > Postman will indicate any [validation errors](/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema/) as you work on your API definition.
 
+### Previewing schema documentation
+
+If you are editing an OpenAPI 2.0 or 3.0 definition,  Postman displays a live preview of your API's schema documentation in the right sidebar. The documentation preview is updated as you edit the definition. You can use the preview to help you visualize your changes as you work.
```
Comment (petewitcher-postman) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md:115]:
```suggestion
If you are editing an OpenAPI 2.0 or 3.0 definition, Postman displays a live preview of your API's schema documentation in the right sidebar. The documentation preview is updated as you edit the definition. You can use the preview to help you visualize your changes as you work.
```

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

**Review by jatin3893 — APPROVED** [review]

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md [hunk 2]
```diff
@@ -97,18 +98,29 @@ To edit an API definition, select an API in the sidebar to expand it, then selec
...
 <img alt="Jumping to a reference location" src="https://assets.postman.com/postman-docs/v10/api-builder-ref-jump-v10.jpg"/>
 
 In the upper right of the API definition editor are options to beautify the content, wrap text, copy, change the file format, and search. When you finish editing your API definition, select **Save**.
 
-<img alt="API Builder editing options" src="https://assets.postman.com/postman-docs/v10/api-builder-edit-options-v10.jpg"/>
+<img alt="API Builder editing options" src="https://assets.postman.com/postman-docs/v10/api-builder-edit-options-v10-21.jpg"/>
 
 > Postman will indicate any [validation errors](/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema/) as you work on your API definition.
 
+### Previewing schema documentation
```
Comment (shashankawasthi88) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md:113]: Can we also add docs for how the users can get into a full screen mode ?

Comment (shashankawasthi88) [reply src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md:113]: Sorry just saw you did mention it. Can we make it more prominent though? I screenshot showing how the full screen view looks like.

**Review by shashankawasthi88 — COMMENTED** [review]
Added a comment

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md [hunk 3]
```diff
@@ -97,18 +98,31 @@ To edit an API definition, select an API in the sidebar to expand it, then selec
...
+
+If you are editing an OpenAPI 2.0 or 3.0 definition, Postman displays a live preview of your API's schema documentation in the right sidebar. The documentation preview is updated as you edit the definition. You can use the preview to help you visualize your changes as you work.
```
Comment (jkonrath-postman) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md:115]: Does this work for OpenAPI 3.1? If so, maybe change this to "OpenAPI 2 or 3"

**Review by jkonrath-postman — COMMENTED** [review]
LGTM

File: src/pages/docs/publishing-your-api/documenting-your-api.md [hunk 4]
```diff
@@ -52,6 +52,8 @@ To view the documentation for an OpenAPI 2.0 or 3.0 API, do the following:
 
 <img alt="Viewing schema documentation" src="https://assets.postman.com/postman-docs/v10/documentation-view-schema-docs-v10-16.jpg" />
 
+> You can also view schema documentation while editing your OpenAPI 2.0 or 3.0 definition. Postman displays a live preview of the documentation as you work on your API. Learn more about [previewing schema documentation](/docs/designing-and-developing-your-api/developing-an-api/defining-an-api/#previewing-schema-documentation).
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/publishing-your-api/documenting-your-api.md:55]: It doesn't work for OpenAPI 3.1

**Review by jatin3893 — APPROVED** [review]

**Pull request merged**